e2e-tests/cypress/e2e/partials/index.js

```diff
@@ -5,19 +5,23 @@ import insurancePartials from './insurance';
 import phaseBanner from './phaseBanner';
 import ukGoodsOrServicesCalculateDescription from './ukGoodsOrServicesCalculateDescription';
 import ukGoodsOrServicesDescription from './ukGoodsAndServicesDescription';
+import yourBusinessSummaryList from './yourBusinessSummaryList';
 
 const partials = {
   backLink: () => cy.get('[data-cy="back-link"]'),
   ukGoodsOrServicesCalculateDescription,
   cookieBanner,
+  errorSummary: () => cy.get('[data-cy="error-summary'),
```

Comment:
I can't see why we need this doesn't seem to be used anywhere, it's also duplicated in `pages/shared`?

Reply:
removed

Reply:
I think we can remove it, it can be done with `errorSummaryListItems` e.g `errorSummaryListItems.length.should('equal', X)`, doing this in other places alreay. `data-cy="error-summary` is the container element and would have to select list items after this selector I think to test it has multiple items?